### ISSUE TITLE
MaterialTextField - Choices - Selects More Than One Option

### ISSUE THREAD

**Issue by AlexandreGrilo** [contrix09/XF-Material-Library#210]
## 🐛 Bug Report

Hello, I am having an issue using `MaterialTextField InputType="Choice"` 
This feature allows you to select more than one option. Returns the last option selected, but visually they are all selected.

### Expected behavior

Since it's a RadioButton, only one option should appear selected.

### Reproduction steps

![emu1](https://user-images.githubusercontent.com/44210508/67854939-51f81100-fb09-11e9-95d2-631a2daf33bd.png)

**XAML**

```
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
             xmlns:material="clr-namespace:XF.Material.Forms.UI;assembly=XF.Material.Forms"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
             mc:Ignorable="d"
             x:Class="MyApp.Page1">
    <ContentPage.Content>
        
        <StackLayout>
            <material:MaterialTextField InputType="Choice"
                                        Placeholder=" - Select - "
                                        Choices="{Binding Options}"/>
        </StackLayout>
    </ContentPage.Content>
</ContentPage>
```

**ViewModel Class**

```
namespace MyApp
{
    public class Page1ViewModel
    {
        private string[] _options;

        public string[] Options
        {
            get { return _options; }
            set { _options = value; }
        }

        public Page1ViewModel ()
        {
            Options = new string[]
            {
                "Option 1",
                "Option 2",
                "Option 3",
                "Option 4"
            };
        }
    }
}
```

**App.xaml**

```
<prism:PrismApplication xmlns="http://xamarin.com/schemas/2014/forms"
                        xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
                        xmlns:d="http://xamarin.com/schemas/2014/forms/design"
                        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
                        xmlns:prism="clr-namespace:Prism.DryIoc;assembly=Prism.DryIoc.Forms"
                        xmlns:mtrl="clr-namespace:XF.Material.Forms.Resources;assembly=XF.Material.Forms"  
                        mc:Ignorable="d"
                        x:Class="MyApp.App">

    <Application.Resources>

        <mtrl:MaterialColorConfiguration x:Key="Material.Color"
                                         Background="#FFFFFF"
                                         Primary="#2c3e50"
                                         Secondary="#1B3147"
                                         PrimaryVariant="#1B3147"
                                         SecondaryVariant="#2c3e50"
                                         Surface="#FFFFFF"/>


        <mtrl:MaterialConfiguration x:Key="Material.Configuration"   
                                    ColorConfiguration="{StaticResource Material.Color}"/>

    </Application.Resources>
</prism:PrismApplication> 
```

**App.cs**

```
protected override async void OnInitialized ()
        {
            InitializeComponent();

            XF.Material.Forms.Material.Init(this, "Material.Configuration");

            await NavigationService.NavigateAsync("/Page1");
        }
```

Thank you all for your attention and if there is any information missing or how can I fix this, please let me know!

### Configuration

Xamarin.Forms 4.2.0.709249
Xamarin.Essentials 1.3.0
XF.Material 1.4.1.3
RG.Plugins.Popup 1.1.5.188
Prism.Core 7.2.0.1367

**Platform:** 

- [x] :robot: Android

**Comment by Miiite** [comment]
I don't reproduce it on the latest version.
Latest available is 1.5.4, and you're using 1.4.1. Can you try updating your library and test it again ?
It that doesn't work, we might need a small repro project

**Comment by AlexandreGrilo** [comment]
Thanks for your reply @Miiite !

I have a test project, with the same version and the feature is working fine. But not in my current project. I don't know why...

So I updated to `version 1.5.4`, it was a little tricky, because I had to install a bunch of `Xamarin.Android.Packages` and I got some errors on my `XAML`. The `assembly` was written
`xmlns:material="clr-namespace:XF.Material.Forms.UI;assembly=XF.Material.Forms"` 
and it should be 
`xmlns:material="clr-namespace:XF.Material.Forms.UI;assembly=XF.Material"`

But now, after that, I've solved the problem, everything works fine. It wasn't because the update properly but something happened here. Black magic hehe

Thank you so much! 

Cheers
